packages/sdk/tests/actions/doesAutoProtocolFeeSharesBuyback.ts

```diff
@@ -0,0 +1,11 @@
+import { type Address, getAbiItem } from "viem";
+import { publicClient } from "../globals.js";
+import { IComptroller } from "../../../abis/src/abis/IComptroller.js";
+
+export function doesAutoProtocolFeeSharesBuyback({ address }: { address: Address }) {
+  return publicClient.readContract({
+    address,
+    abi: [getAbiItem({ abi: IComptroller, name: "doesAutoProtocolFeeSharesBuyback" })],
```

Comment:
You don't need to pre-filter the `abi` here. Just pass `IComptroller` to the `abi` property. Filtering is done based on the `functionName` inside of `readContract` itself.